src/test/java/com/unbappd/enterprise/EnterpriseApplicationTests.java

```diff
@@ -30,9 +31,32 @@ void verifyReviewProperties() {
         reviewEntry.setReviewId(0);
         reviewEntry.setPlacesId(0);
         reviewEntry.setReviewScore(5);
+        reviewEntry.setLongitude(51.1232);
+        reviewEntry.setLongitude(51.1232);
         reviewEntry.setReviewText(reviewTxt);
+        reviewEntry.setReviewerID(101);
 
         assertEquals(reviewTxt, reviewEntry.getReviewText());
         assertEquals(reviewScr, reviewEntry.getReviewScore());
     }
+
+    private void givenReviewDataAreAvailable() {
+    }
+    private void whenUserSearchesWithPlaceID9(){
+        review = reviewService.fetchById(9);
+
+    }
+    private void thenReturnReviewsWithPlaceID9(){
+        String returnedReviewText = review.getReviewText();
+        int checkPlacesID = review.getPlacesId();
+        assertEquals("A fabulous beer", returnedReviewText);
+        assertEquals(9,checkPlacesID);
+    }
+
+    @Test
+    void saveNewReview_validateNewReviewByReturningReviewText() {
```

Comment:
I'm taking a second look at this, please pardon the incorrect name for this. I was initially going to create a different unit test.